### PR TITLE
Create PULL_REQUEST_TEMPLATE.MD

### PR DESCRIPTION
**Pull request by fazerlicourice7**
This is a pull request template for the Mifos X Web App.

### CODE REVIEW

**Comment by mbj36** [comment]
Thanks for the PR, It will be reviewed Shortly!

This message was posted by [GitMate.io](https://gitmate.io) with timestamp signature `Thanks for the PR, It will be reviewed Shortly!:1ePb30:9dWskWDTJ6FT11K5sqFKTVC8p7A`

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Fixes #{Issue Number}
+
+Describe the changes made and why they were made instead of how they were made.
```
Comment (ManrajGrover) [inline .github/PULL_REQUEST_TEMPLATE.MD]: Let's add a header for everything, for example here `## Description`

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Fixes #{Issue Number}
```
Comment (ManrajGrover) [inline .github/PULL_REQUEST_TEMPLATE.MD]: Let's move this below `description` and add a header `## Related issues and discussions` and change `Fixes #{Issue Number}` to `#{Issue Number}`

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+Describe the changes made and why they were made instead of how they were made.
+
+Please Add Screenshots If any UI changes.
```
Comment (ManrajGrover) [inline .github/PULL_REQUEST_TEMPLATE.MD]: Let's add a header called `## Screenshots, if any`

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+Please make sure these boxes are checked before submitting your pull request - thanks!
```
Comment (ManrajGrover) [inline .github/PULL_REQUEST_TEMPLATE.MD]: Let's add a header called `## Checklist` here

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+- [ ] Validate the JS and HTML files with `grunt validate` to detect errors and potential problems in JavaScript code.
+
+- [ ] Run the tests by openining `test/SpecRunner.html` in the browser to make sure you didn't break anything.
```
Comment (ManrajGrover) [inline .github/PULL_REQUEST_TEMPLATE.MD]: Typo here: `openining` -> `opening`

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 6]
```diff
@@ -0,0 +1,18 @@
+##Description
```
Comment (ManrajGrover) [inline .github/PULL_REQUEST_TEMPLATE.MD]: Let's add a space to `##` to fix markdown header ie. `## Description` and not `##Description`

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+Describe the changes made and why they were made instead of how they were made.
+
+##Related issues and discussion
```
Comment (ManrajGrover) [inline .github/PULL_REQUEST_TEMPLATE.MD]: Let's add a space to `##` to fix markdown header like above

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+#{Issue Number}
+
+##Screenshots, if any
```
Comment (ManrajGrover) [inline .github/PULL_REQUEST_TEMPLATE.MD]: Let's add a space to `##` to fix markdown header like above

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+
+##Checklist
```
Comment (ManrajGrover) [inline .github/PULL_REQUEST_TEMPLATE.MD]: Let's add a space to `##` to fix markdown header like above

**Pull request merged**